components/logins/src/logins.udl

```diff
@@ -123,9 +123,6 @@ interface LoginStore {
     [Throws=LoginsApiError]
     boolean delete([ByRef] string id);
 
-    [Throws=LoginsApiError]
```

Comment:
I thought you were considering leaving this in the UDL for a cycle (and a stub impl below)? I'm actually fine if you don't if you are confident it will not cause problems, but if client code *references* this even without actually calling it there might be build errors? Your call though!

Reply:
Yeah that's what I did at first, but thought I'd just open the PR for Android to remove it, then kill it in one go here!